.circleci/config.yml

```diff
@@ -710,6 +730,9 @@ parameters:
   e2e-test-parallelism:
     type: integer
     default: 1
+  scheduled-playwright:
+    type: boolean
+    default: false
```

Comment:
New pipeline parameter to stop launching `build-and-deploy-all-apps-workflow` by CI scheduler.